config.ru

```diff
@@ -30,9 +30,9 @@ end
 
 host = "0.0.0.0"
 bind_addr = "ssl://#{host}?"
-bind_addr << "cert=#{config.ssl_cert}"
-bind_addr << "&key=#{config.ssl_key}"
-bind_addr << "&ca=#{config.ssl_ca_cert}"
+bind_addr << "cert=#{config.ssl_cert}" if config.ssl_cert
```

Comment:
What happens if we don't pass in certs?

Reply:
I think this is leftover from when I wasn't sure if we'd need to pass in a CA cert. Puma SSL doesn't accept self-signed certs with no CA though, so we do need to provide all these. A validation error is raised well before we get here. I'll remove this bit.